models/helper/MarginHelper.py

```diff
@@ -40,10 +40,10 @@ def calculate_margin(buy_size: float = 0.0, buy_filled: int = 0.0, buy_price: in
         sell_value = round(sell_size - sell_fee, 8)
         
         #profit is difference between sell_value and buy_filled in quote currency
-        profit = round(sell_value - buy_filled, 2)
+        profit = round(sell_value - buy_filled, 8)
```

Comment:
it might be out of scope of that PR, but here the precision could be a constant, since its used a lot.

Reply:
Correct, just left this way because we may need more than one constant and need approval from one who has better understanding on the project.

Also we may have a precision config after all, which is a different feature.